app/sources/HFBinaryTemplateRepresenter.m

```diff
@@ -49,16 +49,30 @@ - (void)representerTextView:(HFRepresenterTextView * __unused)sender menu:(NSMen
     if (menu.numberOfItems > 0) {
         [menu addItem:[NSMenuItem separatorItem]];
     }
+    
     self.menuPosition = position;
-    NSString *title = [NSString stringWithFormat:NSLocalizedString(@"Anchor Template at Offset %llu", nil), self.menuPosition];
-    NSMenuItem *menuItem = [[NSMenuItem alloc] initWithTitle:title action:@selector(anchorTemplatesAt:) keyEquivalent:@""];
-    menuItem.target = self;
-    menuItem.enabled = position < self.controller.contentsLength;
-    [menu addItem:menuItem];
+    
+    NSString *anchorTitle = [NSString stringWithFormat:NSLocalizedString(@"Anchor Template at Offset %llu", nil), self.menuPosition];
+    NSMenuItem *anchorMenuItem = [[NSMenuItem alloc] initWithTitle:anchorTitle action:@selector(anchorTemplatesAt:) keyEquivalent:@""];
+    anchorMenuItem.target = self;
+    anchorMenuItem.enabled = position < self.controller.contentsLength;
+    [menu addItem:anchorMenuItem];
+    
+    if ([self.viewController hasTemplate]) {
+        NSString *gotoInTemplateTitle = NSLocalizedString(@"Goto in template", nil);
```

Comment:
"template" should be capitalized.

I think maybe a different name would be more clear. I'll think about it some more. Maybe something like "Show in Template" ... or "Show Matching Template Item"...

Reply:
Changed to "Show in Template" now. Not sure if the item should be disabled instead or hidden?